modules/install/src/main/scala/coursier/install/InstallDir.scala

```diff
@@ -188,24 +196,7 @@ import scala.util.Properties
       case LauncherType.GraalvmNativeImage =>
         assert(appArtifacts.shared.isEmpty) // just in case
 
-        val fetch = simpleFetch(cache, coursierRepositories)
-
-        val graalvmHomeOpt = for {
-          ver <- desc.graalvmOptions.flatMap(_.version)
-            .orElse(graalvmParamsOpt.flatMap(_.defaultVersion))
-          home <- nativeImageJavaHome.map(_(ver).unsafeRun()(cache.ec)) // meh
-        } yield home
-
-        Parameters.NativeImage(mainClass, fetch)
-          .withGraalvmOptions(
-            desc.graalvmOptions.toSeq.flatMap(_.options) ++
-              graalvmParamsOpt.map(_.extraNativeImageOptions).getOrElse(Nil)
-          )
-          .withGraalvmVersion(graalvmParamsOpt.flatMap(_.defaultVersion))
-          .withJars(appArtifacts.fetchResult.files)
-          .withNameOpt(Some(desc.nameOpt.getOrElse(dest.getFileName.toString)))
-          .withJavaHome(graalvmHomeOpt)
-          .withVerbosity(verbosity)
+        bootstrapParamsLike(desc, appArtifacts, infoEntries, mainClass, baseJarPreamble0)
```

Comment:
I'm not sure we should fallback to JVM… Let's just fail rather.

Reply:
If we throw an error he, then how can user install application that isn't native version available for his architecture, but only the JVM version is available?